src/account/selectors.ts

```diff
@@ -47,8 +43,6 @@ export const recoveringFromStoreWipeSelector = (state: RootState) =>
 export const accountToRecoverSelector = (state: RootState) =>
   state.account.accountToRecoverFromStoreWipe
 
-export const backupCompletedSelector = (state: RootState) => state.account.backupCompleted
```

Comment:
i see this is duplicated in the src/backup folder, but the state is in `account`...so what do you think about keeping this one and removing the other one instead?

Reply:
Ahh good catch! We can delete that other file entirely now.